### DISCUSSION TITLE
I got different result though " model, alphabet = torch.hub.load("facebookresearch/esm:main", "esm1b_t33_650M_UR50S") " and "model, alphabet = esm.pretrained.esm1b_t33_650M_UR50S()"

### DISCUSSION THREAD

**Discussion by Ming-Qin-tech** [facebookresearch/esm#208] [Q&A]
when using the two different ways to load the model, I got a different result when trying to predict the mask token in the same sequence.

**Comment by tomsercu** [comment]
Those two methods give you the same model.
Potentially you didn't set the model in eval mode? In which case the dropout makes the results non-deterministic. 
see eg https://github.com/facebookresearch/esm/blob/main/scripts/extract.py#L64
